pkg/gui/filetree/node.go

```diff
@@ -300,3 +302,10 @@ func (self *Node[T]) ID() string {
 func (self *Node[T]) Description() string {
 	return self.GetPath()
 }
+
+func (self *Node[T]) Name() string {
+	if self.IsFile() {
+		return path.Base(self.Path)
+	}
```

Comment:
Why do this only for files? I would expect to be able to copy the name of a directory too, and get just the name. In other words, I'd just always `return path.Base(self.Path)`, unconditionally.